snuba/datasets/storages/factory.py

```diff
@@ -48,3 +48,9 @@ def get_writable_storage(storage_key: str) -> WritableTableStorage:
 
 def get_cdc_storage(storage_key: str) -> CdcStorage:
     return CDC_STORAGES[storage_key]
+
+def get_storage_key(storage: ReadableTableStorage) -> str:
```

Comment:
I suppose this is all temporary since the storage name is likely moving into the storage.
Perhaps though it is better to just go ahead and make subscriptions and confirm_load take a storage instead of dataset anyway.

Reply:
Seems this would be a significantly larger change, not going to make it right now. I think we should try to get DEFAULT_DATASET_BROKERS deprecated first so we can stop requiring users to provide both bits of configuration as soon as possible.

Reply:
Deleted, key is in the storage now